File based role mappings vs the role mapping APIs (#47015)
elastic/elasticsearch: x-pack/docs/en/rest-api/security/get-role-mappings.asciidoc

==== {api-description-title}

Role mappings define which roles are assigned to each user. For more information, 
see <<mapping-roles>>.

> The role mapping APIs are generally the preferred way to manage role mappings

Reviewer: ```suggestion
The <<security-api-put-role-mapping>> is the preferred way to manage role mappings
```

Author: This talks about all the related APIs ( get/put/delete) instead of simply the put one
